### PR TITLE
Feature/792 inertial filter

### PR DESCRIPTION
**Pull request by thibaudteil**
* **Tickets addressed:** bsk-792
* **Review:** By commit 
* **Merge strategy:** Merge (no squash) 
 
## Description
Inertial attitude filter very similar to the C implementation which uses star tracker measurements and gyro measurements to estimate inertial attitude. Filter uses the SRuKF interface class

## Verification
Unit test

## Documentation
Rst documentation

## Future work
None

### CODE REVIEW

File: src/fswAlgorithms/attDetermination/inertialAttitudeUkf/inertialAttitudeUkf.cpp
Comment (patkenneally) [inline src/fswAlgorithms/attDetermination/inertialAttitudeUkf/inertialAttitudeUkf.cpp]: :large_orange_diamond: These lines could become `auto measurement = this->measurements[index].value()`

File: src/fswAlgorithms/attDetermination/inertialAttitudeUkf/inertialAttitudeUkf.cpp
Comment (patkenneally) [inline src/fswAlgorithms/attDetermination/inertialAttitudeUkf/inertialAttitudeUkf.cpp]: :large_blue_circle: This brace is over indented.

File: src/fswAlgorithms/attDetermination/inertialAttitudeUkf/inertialAttitudeUkf.cpp
Comment (patkenneally) [inline src/fswAlgorithms/attDetermination/inertialAttitudeUkf/inertialAttitudeUkf.cpp]: :large_blue_circle: This brace is over indented.

File: src/fswAlgorithms/attDetermination/inertialAttitudeUkf/inertialAttitudeUkf.cpp
Comment (patkenneally) [inline src/fswAlgorithms/attDetermination/inertialAttitudeUkf/inertialAttitudeUkf.cpp]: :large_blue_circle: This could be declared and initialized in one expression using `Eigen::Vector3d wheelTorque = Eigen::Vector3d::Zero();`

File: src/fswAlgorithms/attDetermination/inertialAttitudeUkf/tests/test_inertialAttitudeUkf.py
Comment (patkenneally) [inline src/fswAlgorithms/attDetermination/inertialAttitudeUkf/tests/test_inertialAttitudeUkf.py]: :large_orange_diamond: This indirection can be removed. Rename the called function `test_` etc.

Same for the above test function

**Review by patkenneally — APPROVED** [review]
Legend:
:small_red_triangle_down: *Issues* to address before merge
:large_orange_diamond: *Requests* that should not block merge, but should at least be discussed
:large_blue_circle: *Recommendations* that can be ignored if desired

File: src/fswAlgorithms/attDetermination/inertialAttitudeUkf/inertialAttitudeUkf.cpp
Comment (thibaudteil) [reply src/fswAlgorithms/attDetermination/inertialAttitudeUkf/inertialAttitudeUkf.cpp]: Done

File: src/fswAlgorithms/attDetermination/inertialAttitudeUkf/inertialAttitudeUkf.cpp
Comment (thibaudteil) [reply src/fswAlgorithms/attDetermination/inertialAttitudeUkf/inertialAttitudeUkf.cpp]: Fixed

File: src/fswAlgorithms/attDetermination/inertialAttitudeUkf/inertialAttitudeUkf.cpp
Comment (thibaudteil) [reply src/fswAlgorithms/attDetermination/inertialAttitudeUkf/inertialAttitudeUkf.cpp]: The whole method was over indented! Fixed

File: src/fswAlgorithms/attDetermination/inertialAttitudeUkf/inertialAttitudeUkf.cpp
Comment (thibaudteil) [reply src/fswAlgorithms/attDetermination/inertialAttitudeUkf/inertialAttitudeUkf.cpp]: Done

File: src/fswAlgorithms/attDetermination/inertialAttitudeUkf/tests/test_inertialAttitudeUkf.py
Comment (thibaudteil) [reply src/fswAlgorithms/attDetermination/inertialAttitudeUkf/tests/test_inertialAttitudeUkf.py]: done

**Pull request merged**
